flare/kernels/map_3b_kernel.py

```diff
@@ -0,0 +1,654 @@
+#                     if (ei2 == c2):
+#                         r13 = ri1 - rj3
+#
+#                     rj2 = bond_array_1[ind1, 0]
+#                     if (ei1 == ej2):
+#                         r21 = ri2 - rj1
+#                     if (ei1 == ej1):
+#                         r22 = ri2 - rj2
+#                     if (ei1 == c2):
+#                         r23 = ri2 - rj3
+#                     cj2 = bond_array_1[ind1, d1]
+#                     fj2, _ = cutoff_func(r_cut, rj2, 0)
+#                     # del ri2
+#
+#                     rj3 = cross_bond_dists_1[m, m + n + 1]
+#                     if (c1 == ej2):
+#                         r31 = ri3 - rj1
+#                     if (c1 == ej1):
+#                         r32 = ri3 - rj2
+#                     if (c1 == c2):
+#                         r33 = ri3 - rj3
+#                     fj3, _ = cutoff_func(r_cut, rj3, 0)
+#                     # del ri3
+#
+#                     fj = fj1 * fj2 * fj3
+#                     # del fj1
+#                     # del fj2
+#                     # del fj3
+#
+#                     if (c1 == c2):
+#                         if (ei1 == ej1) and (ei2 == ej2):
+#                             kern += three_body_en_helper(ci1, ci2, r11, r22,
+#                                                          r33, fi, fj, fdi, ls1,
+#                                                          ls2, sig2)
+#                         if (ei1 == ej2) and (ei2 == ej1):
+#                             kern += three_body_en_helper(ci1, ci2, r12, r21,
+#                                                          r33, fi, fj, fdi, ls1,
+#                                                          ls2, sig2)
+#                     if (c1 == ej1):
+#                         if (ei1 == ej2) and (ei2 == c2):
+#                             kern += three_body_en_helper(ci1, ci2, r13, r21,
+#                                                          r32, fi, fj, fdi, ls1,
+#                                                          ls2, sig2)
+#                         if (ei1 == c2) and (ei2 == ej2):
+#                             kern += three_body_en_helper(ci1, ci2, r11, r23,
+#                                                          r32, fi, fj, fdi, ls1,
+#                                                          ls2, sig2)
+#                     if (c1 == ej2):
+#                         if (ei1 == ej1) and (ei2 == c2):
+#                             kern += three_body_en_helper(ci1, ci2, r13, r22,
+#                                                          r31, fi, fj, fdi, ls1,
+#                                                          ls2, sig2)
+#                         if (ei1 == c2) and (ei2 == ej1):
+#                             kern += three_body_en_helper(ci1, ci2, r12, r23,
+#                                                          r31, fi, fj, fdi, ls1,
+#                                                          ls2, sig2)
+#     return kern
```

Comment:
Can this comment block be removed?

Reply:
It's left there for the next step (force mapping). It'll be revived in the factory branch